Tasks/PublishSymbols/Publish-Symbols.ps1

```diff
@@ -0,0 +1,284 @@
+<#
+.SYNOPSIS
+Script to publish a symbol request to Artifact Store Symbol Service
+
+.PARAMETER RequestName 
+The name to give the published request.
+
+.PARAMETER SourcePath
+The root directory containing the files to be analyzed for inclusion in the request. Only valid debug files will actually be included in the published request.
+
+.PARAMETER SymbolServiceUri
+The VSTS Symbol Service to which the request should be published.
+
+.PARAMETER ExpirationInDays
+The number of days that symbols should be retained.  If not specified, the default settings of the service will be used.
+
+.PARAMETER PersonalAccessToken
+Optional. Use the provided PAT to authenticate to the Symbol Service.  If not provided, AAD authentication will be used.
+
+.EXAMPLE
+Publish-Symbols -SymbolServiceUri "https://microsoft.artifacts.visualstudio.com/defaultcollection" -RequestName $(Build.BuildNumber) -SourcePath $(Agent.BuildDirectory)\bin
+#>
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory=$true)]
+    [string] $SymbolServiceUri,
+
+    [Parameter(Mandatory=$true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $RequestName,
+
+    [Parameter(Mandatory=$true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $SourcePath,
+
+    [Parameter(Mandatory=$false)]
+    [string] $ExpirationInDays,
+
+    [Parameter(Mandatory=$false)]
+    [string] $PersonalAccessToken
+)
+
+# -----------------------------------------------------------------------------
+# Methods
+# -----------------------------------------------------------------------------
+function Download-SymbolClient([string]$symbolServiceUri, [string]$directory)
```

Comment:
have you considered bundling this tool with your task? that would eliminate additional back-compat hurdles.